lib/tapioca/helpers/cli_helper.rb

```diff
@@ -8,7 +8,7 @@ module CliHelper
     extend T::Sig
     extend T::Helpers
 
-    requires_ancestor { Thor::Shell }
+    include Thor::Base
```

Comment:
It doesn't make sense to include `Base` here. This module is a helper that (for now) only requires the client to bring `Shell`. It shouldn't bring [options](https://github.com/rails/thor/blob/master/lib/thor/base.rb#L35) nor anything related to command handling.